src/plugins/unified_search/public/filter_bar/filter_editor/value_input_type.tsx

```diff
@@ -26,6 +26,16 @@ interface Props {
 }
 
 class ValueInputTypeUI extends Component<Props> {
+  private getValueForNumberField = (value?: string | number) => {
```

Comment:
Could we add an explicit return type?
```suggestion
  private getValueForNumberField = (value?: string | number): undefined | string | number => {
```